src/content/keynoters/documentary.mdx

```diff
@@ -0,0 +1,7 @@
+---
+name: "Python: the Documentary" screening and Q&A
+image: ./cultrepo.png
+slug: documentary-and-q-and-a
+tagline: Keynote panel and screening of CultRepo (formerly Honeypot)'s "Python: the Documentary"
```

Comment:
Perhaps? To avoid the apostrophe after parentheses.
```suggestion
tagline: Keynote panel and screening of "Python: the Documentary" by CultRepo (formerly Honeypot)
```